spec/features/create_review_spec.rb

```diff
@@ -8,10 +8,15 @@
 
     end
 
-    scenario "user properly enters a new review"  do
-      weather = FactoryGirl.create(:weather)
+    scenario "user properly enters a new review" , %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Space found before comma.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.